src/operator/exhaustMap.ts

```diff
@@ -6,35 +6,33 @@ import {errorObject} from '../util/errorObject';
 import {OuterSubscriber} from '../OuterSubscriber';
 import {subscribeToResult} from '../util/subscribeToResult';
 
-export function exhaustMap<T, R, R2>(project: (value: T, index: number) => Observable<R>,
-                                     resultSelector?: (outerValue: T,
-                                                       innerValue: R,
-                                                       outerIndex: number,
-                                                       innerIndex: number) => R2): Observable<R> {
+export function exhaustMap<T, R, TResult>(project: (value: T, index: number) => Observable<R>,
```

Comment:
I can do that, but do you mind if I ask why specificly (outside keeping it like it was).  I just personally found `R` and `R2` fairly confusing looking at the code.